lua/autorun/sh_cami.lua

```diff
@@ -357,11 +357,14 @@ CAMI.PlayerHasAccess
 ]]
 -- Default access handler
 local defaultAccessHandler = {["CAMI.PlayerHasAccess"] =
-    function(_, actorPly, privilegeName, callback, _, extraInfoTbl)
+    function(_, actorPly, privilegeName, callback, targetPly, extraInfoTbl)
         -- The server always has access in the fallback
         if not IsValid(actorPly) then return callback(true, "Fallback.") end
 
         local priv = privileges[privilegeName]
+        if priv.HasAccess then
+            return callback(priv:HasAccess(actorPly, targetPly), "Fallback.")
```

Comment:
I assumed that `HasAccess` would be called _after_ `MinAccess` was checked rather than before it.
eg if the priv is for kicking people and the `HasAcess` check wants to make sure an admin isn't trying to kick a superadmin.